docs/en/src/context.md

```diff
@@ -1 +1,21 @@
 # Context
+
+The main goal of `Context` is to acquire global data attached to Schema. **Note that if the return value of your Resulve boored data in `Context`, you need to explictly state the lifetime of the argument.**
```

Comment:
Note that if the return value of your resolve function is borrowed from `Context`, you need to explicitly state the lifetime of the argument.